apps/builder/app/builder/features/props-panel/props-panel.stories.tsx

```diff
@@ -40,6 +42,13 @@ pagesStore.set({
   ],
 });
 
+const urlSections = new Map([
+  [unique(), "links"],
+  [unique(), "about"],
+]);
+
+setMockUrlSectionsStore(atom(urlSections));
```

Comment:
I see no problem in mocking writable atoms. We would have testing element here.

I usually add createInstancePair and createPropPair utilities to make formatting better and not duplicate id

```ts
pagesStore.set({
  homePage: { id: 'home', rootInstanceId: 'root' }
  pages: []
})
instances.set(new Map([
  ['root', {
    id: 'root',
    component: 'Box',
    children: []
  }]
]))
props.set(new Map([
  ['prop1', {
    type: "string",
    name: "id",
    value: "links",
    instanceId: 'root'
  }],
  ['prop3', {
    type: "string",
    name: "id",
    value: "about",
    instanceId: 'root'
  }],
]))
```